app/packet_handlers.py

```diff
@@ -1309,6 +1384,12 @@ async def user_leaves_channel_handler(session: "Session", packet_data: bytes):
     channel = await channels.fetch_one_by_name(channel_name)
     if channel is None:
         return
+    
+    # NOTE: we ignore #lobby to enqueue the match updates
+    # and we actually remove them from the channel on lobby part
+    presence: Presence | None = session.get("presence")
```

Comment:
is there a reason get is used? presence should never be None

Reply:
yeah this is true for now, will adjust